sickchill/oldbeard/generic_queue.py

```
@@ -82,6 +82,41 @@ def promote_item(self, item):
+            item.join(timeout)
+            if item.is_alive():
+                logger.warning(f"{name} did not exit within {timeout:g}s; continuing shutdown")
+            else:
+                logger.info(f"{name} exited")
+
+        with self.lock:
+            if self.currentItem is item:
+                try:
+                    item.finish()
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)